### PR TITLE
Update benchmarks

### PR DESCRIPTION
**Pull request by naliboff**
This PR is on top of the single commit in #3372 and is part of the preparation for #3306. 

The changes to the benchmarks, and one corresponding test, either remove boundary effects in the compositional fields for viscoelastic stresses or remove viscoelastic effects in purely viscous material. 

This should produce a change in the test results, but the benchmarks have already been rerun to confirm improved results.

### CODE REVIEW

File: tests/viscoelastic_plastic_brick_extension.prm [hunk 1]
```diff
@@ -35,6 +35,31 @@ subsection Initial composition model
   end
 end
 
+# Use viscoelastic plastic material model
+subsection Material model
```
Comment (tjhei) [inline tests/viscoelastic_plastic_brick_extension.prm]: When you later update the base (included prm), we should probably remove the material model section here.

**Comment by tjhei** [comment]
Excellent, looks good.

File: tests/viscoelastic_plastic_brick_extension.prm (hunk 1 above)
Comment (naliboff) [reply tests/viscoelastic_plastic_brick_extension.prm]: That is the plan 👍. The `viscoelastic_plastic` test suite will keep the `Material model` section as the corresponding benchmarks will soon use `visco_plastic`, but the intention is to completely remove `viscoelastic_plastic` (source code, tests) soon thereafter.

Comment (naliboff) [reply tests/viscoelastic_plastic_brick_extension.prm]: Thanks for handling all of the reviews and merges today!

**Pull request merged**